examples/guestbook/README.md

```diff
@@ -436,8 +436,8 @@ If you are in a live kubernetes cluster, you can just kill the pods, using a scr
 
 ```shell
 ### First, kill services and controllers.
-kubectl stop rc examples/guestbook/redis-slave-controller.json
-kubectl stop rc examples/guestbook/frontend-controller.json
+kubectl stop rc examples/guestbook/redis-slave-controller
+kubectl stop rc examples/guestbook/frontend-controller
```

Comment:
Is it necessary to add ``-f``? On the master of my ubuntu cluster, I can stop those two rc without ``-f``